carbonmark-api/postman_tests.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Start the server in the background
+npm run start:test &
+
+# Get the process ID of the server
+server_pid=$!
+
+# Wait for the server to be available
+while ! curl -s http://localhost:3000 > /dev/null; do
+  sleep 1
+done
+
+# Run the Newman tests
+npx newman run postman_collection.json
```

Comment:
Amazing, I've never written tests with postman, I think i'll call on you to give a little workshop for us 👑